docs/content/integrations/airflow.mdx

```diff
@@ -9,3 +9,268 @@ Migrating from Airflow to Dagster, or integrating Dagster into your existing wor
+      <td>
+        <a href="/deployment/dagster-instance">Instances</a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Operators</td>
+      <td>None</td>
+      <td>
+        Dagster uses normal Python functions instead of framework-specific
+        operator classes. For off-the-shelf functionality with third-party
+        tools, Dagster provides{" "}
+        <a href="/integrations">integration libraries</a>.
+      </td>
+    </tr>
+    <tr>
+      <td>Pools</td>
+      <td>
+        <a href="/deployment/run-coordinator">Run coordinators</a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Plugins/Providers</td>
+      <td>
+        <a href="/integrations">Integrations</a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Schedulers</td>
+      <td>
+        <a href="/concepts/automation/schedules">Schedules</a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Sensors</td>
+      <td>
+        <a href="/concepts/partitions-schedules-sensors/sensors">Sensors</a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>SubDAGs/TaskGroups</td>
+      <td>
+        <ul style={{ marginTop: "0em" }}>
+          <li style={{ marginTop: "0em" }}>
+            <a href="/concepts/ops-jobs-graphs/graphs">Graphs</a>
+          </li>
+          <li>
+            <a href="/concepts/metadata-tags/tags">Tags</a>
+          </li>
+          <li>
+            <a href="/concepts/assets/software-defined-assets#grouping-assets">
+              Asset groups
+            </a>
+          </li>
+        </ul>
+      </td>
+      <td>
+        Dagster provides rich, searchable{" "}
+        <a href="/concepts/metadata-tags">
+          metadata and <a href="/concepts/metadata-tags/tags">tagging</a>
+        </a>{" "}
+        support beyond what’s offered by Airflow.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>task_concurrency</code>
+      </td>
+      <td>
+        <a href="/guides/limiting-concurrency-in-data-pipelines#limiting-opasset-concurrency-across-runs">
+          Asset/op-level concurrency limits
+        </a>
+      </td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Trigger</td>
+      <td>
+        <a href="/concepts/webserver/ui">Dagster UI Launchpad</a>
+      </td>
+      <td>
+        Triggering and configuring ad-hoc runs is easier in Dagster which allows
+        them to be initiated through the{" "}
+        <a href="/concepts/webserver/ui">Dagster UI</a>, the{" "}
+        <a href="/concepts/webserver/graphql">GraphQL API</a>, or the CLI.
+      </td>
+    </tr>
+    <tr>
+      <td>XComs</td>
+      <td>
+        <a href="/concepts/io-management/io-managers">I/O managers</a>
+      </td>
+      <td>
+        I/O managers are more powerful than XComs and allow the passing large
+        datasets between jobs.
+      </td>
+    </tr>
+  </tbody>
+</table>
+>>>>>>> bd51840130 ([docs] 4/n dagster as single-pane-of-glass for organization workflows)
```

Comment:
Should this line be removed? It seems to have been forgotten when resolving a Git conflict.

Reply:
Yikes, good catch.